ckcc/cli.py

```diff
@@ -1010,4 +1006,70 @@ def get_storage_locker():
 
     click.echo(ls)
 
+
+@main.command('coldcardify')
+@click.argument('file', type=click.Path(exists=True), required=True)
+@click.option('--outfile', '-o', type=click.Path(),
+              help="output file path where adjusted wallet file is written. "
+                   "If this is not specified output is written to <original_file>_cc")
+@click.option('--dry-run', '-n', default=False, is_flag=True, help="do not write files instead pretty print to console")
+def electrum_coldcardify(file, outfile, dry_run):
+    """
+    Coldcardify electrum wallet file.
+
+    Adjusts standard hardware electrum wallet file to work with Coldcard. Under the hood this command changes
+    values in keystore dict in electrum wallet file. 'hw_type' is changed to coldcard. 'soft_device_id' is set to null.
+    'ckcc_xpub' is added (if coldcard is connected). And 'label' is built using 'Coldcard' + master fingerprint.
+
+    Your Coldcard does not need to be connected, but it is better to have it connected because it can be checked whether
+    Coldcard and wallet file describe the same wallet.
+
+    If no '--outfile/-o' is specified, new wallet file will be created in same location with '_cc' suffix.
+
+    Rationale:
+      Users may want to switch their hardware wallet vendor. Yet they want to keep all of their electrum data
+      (labels, contacts, payment requests...). Another possibility is when someone's hww gets lost or broken,
+      and they still have a seed, this seed can be loaded to new Coldcard (check https://coldcard.com/docs/import)
+      and their previous electrum wallet can be coldcardified.
+
+    * Does not work with encrypted wallet files - please decrypt first via electrum interface.
+    """
+    if file == outfile:
+        click.echo("'FILE' and '--outfile' cannot be the same")
+        sys.exit(1)
+    try:
+        # this may be changed to context manager once https://github.com/Coldcard/ckcc-protocol/pull/14 gets merged
+        dev = get_device()
+    except KeyError:
+        dev = None
+
+    try:
+        # open file only for reading and close it immediately after it is loaded into memory
+        with open(file, "r") as f:
+            contents = json.loads(f.read())
+    except Exception as e:
+        click.echo("Failed to load wallet file: {}".format(e))
+        sys.exit(1)
+
+    # for now only standard wallet
+    assert contents["wallet_type"] == "standard", "Not a standard wallet"
+    # below line adjust keystore dict in contents dict (in place)
+    cc_adjust_hww_keystore(contents["keystore"], dev)
```

Comment:
IIRC multisig wallets have multiple "keystore" entries, but should otherwise be possible to convert